### PR TITLE
Add basic readme

### LINKED ISSUE #121: Write up a decent README
Some things that should be covered:
* Link to the matrix channel
* Deployment guide

### PR DESCRIPTION
**Pull request by robbevp**
Fix #121 
Add a basic readme.

#### Topics to cover
* [x] Info about app (quick first draft)
* [x] Deployment guide @chvp can you provide this?
* [x] Info for local development
* [x] Info for questions/bugs

### CODE REVIEW

**Review by chvp — APPROVED** [review]
@robbevp Can you look over my changes? I can't request your review since you created the PR.

File: README.md [hunk 1]
```diff
@@ -1,24 +1,74 @@
...
-* Database creation
+1. Make sure you have the correct dependencies installed. This is
+   mostly ffmpeg and ruby.
+1. Create a checkout of the code.
+1. Install all ruby dependencies with `bundle install`.
+1. Setup the database. Only postgresql is supported. Don't forget to
+   run `rails db:setup`.
+1. Make sure to perform migrations before starting the server. Also
+   run the `ffmpeg:check_version` rake task.
+1. Run `puma -C config/puma.rb` to start the server. You probably want
```
Comment (robbevp) [inline README.md:31]: Should we mention the other options (Passenger) as well?

File: README.md [hunk 2]
```diff
@@ -1,24 +1,74 @@
...
+   to set the following environment variables:
+    * DATABASE_URL
```
Comment (robbevp) [inline README.md:33]: I would split this list between those that should be set, and the optional ones. the DATABASE_URL and RAILS_ENV=production seem non-optional to me

File: README.md (hunk 1 above)
Comment (chvp) [reply README.md]: Done. (The other item as well.)

**Comment by robbevp** [comment]
@chvp I can't approve, but LGTM

**Pull request merged**